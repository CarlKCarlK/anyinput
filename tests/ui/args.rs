use anyinput::anyinput;

#[anyinput(not_empty)]
pub fn any_str_len(s: AnyIter(AnyString)) -> Result<usize, anyhow::Error> {
    let len = s.len();
    Ok(len)
}

fn main() {}
