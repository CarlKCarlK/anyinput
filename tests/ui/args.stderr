error: anyinput does not take any arguments.
 --> tests/ui/args.rs:3:12
  |
3 | #[anyinput(not_empty)]
  |            ^^^^^^^^^
